subprojects/libnvme.wrap

```diff
@@ -1,6 +1,6 @@
 [wrap-git]
 url = https://github.com/linux-nvme/libnvme.git
-revision = c90eeff5f5bfab597a6152d979aa7049524dd79f
+revision = b8ea8bee138af7a336874b47c160fed91c2b1969
```

Comment:
I've applied the MOS/MO fixes, the reference has changed. Could you please update this accordingly? Thanks!

Reply:
@igaw  Thanks for the fast response, rebased and modified with merged hash :)